lucet-spectest/src/lib.rs

```diff
@@ -42,19 +41,15 @@ pub fn run_spec_test(spec_path: &PathBuf) -> Result<SpecScriptResult, Error> {
     Ok(res)
 }
 
-fn unexpected_failure(e: ScriptError) -> SpecTestError {
+fn unexpected_failure(e: ScriptError) -> Error {
     if e.unsupported() {
-        Error::from(e)
-            .context(SpecTestErrorKind::UnsupportedLucetc)
-            .into()
+        Error::UnsupportedLucetc
     } else {
-        Error::from(e)
-            .context(SpecTestErrorKind::UnexpectedFailure)
-            .into()
+        Error::UnexpectedFailure(String::new())
```

Comment:
:thinking: :thought_balloon: idle thought: should this `String::new` be here? Wondering if either (a) `Error::UnexpectedFailure` doesn't need a payload, or if we could do something in the spirit of `e.to_string()`.

Reply:
Thanks for the comment!  I've made this change.